api/src/campaigns/schemas/campaignList.schema.ts

```diff
@@ -0,0 +1,20 @@
+import { isState } from 'src/shared/validations/isState'
+import { z } from 'zod'
+
+export const campaignListSchema = z.object({
+  id: z.coerce.number().optional(),
+  state: isState()
+    .transform((val) => val.toUpperCase())
+    .optional(),
+  email: z.string().email().optional(),
+  slug: z.string().optional(),
+  level: z.enum(['LOCAL', 'CITY', 'COUNTY', 'STATE', 'FEDERAL']).optional(),
+  primaryElectionDateStart: z.string().date().optional(),
+  primaryElectionDateEnd: z.string().date().optional(),
+  campaignStatus: z.enum(['active', 'inactive']).optional(),
```

Comment:
Good to put any string literal into a constant, just in case it gets used elsewhere and then can be managed in just one place.